assets/src/dashboard/app/views/myStories/content/index.js

```diff
@@ -31,7 +30,6 @@ import {
 /**
  * Internal dependencies
  */
-import { TransformProvider } from '../../../../../edit-story/components/transform';
```

Comment:
I particularly love getting rid of this dependency on `edit-story` 🎉

Reply:
I'm not sure yet what to do about templates, but this will at least help with my stories.

Reply:
We should probably do that for templates too. We could host the preview images on the CDN.

Reply:
It's a matter of how we're generating the preview images, since the related image rendering for thumbnails is still being researched.

Reply:
For templates, I imagine poster generation is a matter of flattening/rasterizing the first page after removing [elements that are duplicative with the story preview](https://developers.google.com/search/docs/advanced/appearance/enable-web-stories#google-discover) (i.e. title text/author/publisher logo). For example:

Before:
![Screen Shot 2021-07-13 at 3 25 25 PM](https://user-images.githubusercontent.com/20525523/125512944-241d8986-31bd-42bd-bf67-9f36a5debeec.png)

After (removed title, author, logo):
![Screen Shot 2021-07-13 at 3 25 37 PM](https://user-images.githubusercontent.com/20525523/125512977-ad8dae68-9a7d-4497-8934-f7df37d086cf.png)

In many cases this might just be the background image of the first page, but some templates have stickers/scrims that may still be desired. I took screenshots of the editor above, but actual posters should be exported from Figma.

/cc @o-fernandez

Reply:
i suppose if we want to just export this from figma we could do this pretty quickly.

Reply:
@choumx @swissspidy following up on this, should we just export from figma and add asset paths to the templates for poster images and use those here or should we put a pin in this until later when we deal with text sets and page layouts?